pkg/clusteragent/admission/mutate/tags.go

```diff
@@ -46,41 +48,52 @@ func InjectTags(req *admiv1beta1.AdmissionRequest, dc dynamic.Interface) (*admiv
 // env vars into a pod template if needed
 func injectTags(pod *corev1.Pod, ns string, dc dynamic.Interface) error {
 	if pod == nil {
+		metrics.MutationAttempts.Inc(metrics.TagsMutationType, strconv.FormatBool(false))
```

Comment:
It might be possible to put that as a `defer` using a closure that uses value from a `var bool injected` declared at the top.
I believe it would be a bit clearer/easier to read.

Reply:
good point! thanks

Reply:
changes made 👍